llvm/lib/Transforms/Utils/AMDGPUEmitPrintf.cpp

```diff
@@ -170,20 +173,46 @@ static Value *appendString(IRBuilder<> &Builder, Value *Desc, Value *Arg,
   return callAppendStringN(Builder, Desc, Arg, Length, IsLast);
 }
 
+static Value *appendVectorArg(IRBuilder<> &Builder, Value *Desc, Value *Arg,
```

Comment:
All of the codegen changes here should be a separate commit

Reply:
Done

Reply:
These are all still in this PR?